rust/agama-lib/src/users/settings.rs

```diff
@@ -63,3 +83,9 @@ pub struct RootUserSettings {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub ssh_public_key: Option<String>,
 }
+
+impl RootUserSettings {
+    pub fn skip_export(&self) -> bool {
+        self.password.is_none() && self.hashed_password.is_none() && self.ssh_public_key.is_none()
```

Comment:
@imobachgs sadly here we have identical problem to first user with hashed password...but here it is even more tricky as password and hashed_password are always skipped during deserializing, so unless ssh_public_key is set, then it is always empty....but on other hand we want somehow indicate user that they need to set it in root user.
So I am not sure how to proceed here. Of course I can change it to something like `self.hashed_password == Some(false)`

Reply:
btw if hashed_password is set to true, should we really omit serialization of password to profile?